src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,47 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int NUMBER_OF_ELEMENTS = 10;
+    private Object[] keys;
+    private Object[] values;
+
+    public StorageImpl() {
+        this.keys = new Object[NUMBER_OF_ELEMENTS];
+        this.values = new Object[NUMBER_OF_ELEMENTS];
+    }
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < NUMBER_OF_ELEMENTS; i++) {
+            if (keys[i] == null && values[i] == null
+                    || keys[i] == key && values[i] != null
+                    || keys[i] != null && keys[i].equals(key)) {
+                keys[i] = key;
+                values[i] = value;
+                break;
+            }
+        }
     }
 
     @Override
     public V get(K key) {
+        for (int i = 0; i < keys.length; i++) {
+            if (keys[i] == key || (keys[i] != null && keys[i].equals(key))) {
+                return (V) values[i];
+            }
+        }
         return null;
     }
 
     @Override
     public int size() {
-        return -1;
+        int size = 0;
+        for (int i = 0; i < keys.length; i++) {
+            if (keys[i] == null && values[i] == null) {
+                size = i;
+                break;
+            }
+        }
+        return size;
```

Comment:
```suggestion
        return size;
```
lets add a field in the storage to store the actual size
increment the size when adding a new pair